### PR TITLE
Updated README for istio.

### PR DESCRIPTION
**Pull request by gyliu513**
/cc @lachie83

### CODE REVIEW

**Comment by gyliu513** [comment]
ping @lachie83

File: incubator/istio/README.md [hunk 1]
```diff
@@ -148,5 +148,5 @@ These addons can be selectively installed by setting `addons.<addon-name>.enable
 If you are running a Kubernetes 1.7+ and have the Initializers api enabled you may choose to enable the Initializer to be installed. See the [docs](https://kubernetes.io/docs/admin/extensible-admission-controllers/) on how to enable.
 
 ```console
-helm install --name istio . --namespace istio-system --set istio.install=true,initializer.enabled=true
+helm install --name my-release incubator/istio --namespace istio-system --set istio.install=true,initializer.enabled=true
```
Comment (lachie83) [inline incubator/istio/README.md]: Can you also add the --devel flag until we get the chart version sorted out.

Comment (gyliu513) [reply incubator/istio/README.md]: Done

**Comment by gyliu513** [comment]
Thanks @lachie83 sorry for the late update.

**Comment by lachie83** [comment]
Can you rebase on master so the linter passes?

**Comment by gyliu513** [comment]
@lachie83 lint works well, am I missing anything?

```
root@k8s001:~/go/src/github.com/gyliu513/charts# helm lint incubator/istio
==> Linting incubator/istio
Lint OK

1 chart(s) linted, no failures
```

**Comment by mattfarina** [comment]
@gyliu513 The linter does more than helm lint. See the detailed output next to the failed CircleCI. In this case, the version for the chart isn't incremented past what's on master. That's automatically checked now.

**Comment by gyliu513** [comment]
@mattfarina so you mean I need to update the version of the chart each time I update it? If so, this does not seem reasonable, as sometimes, it maybe just a trial/small/nit fix and may not deserve a version update.

**Comment by mattfarina** [comment]
@gyliu513 The master branch is continuously deployed for the community repo. Every change to a chart increments the version here. If you want to debate that you'll need to make a proposal for a new strategy that works with the continuous automation employed. That would be separate from this PR.

For this PR to be merged you'll need to increment the version number (in this case you could increment the patch level).

_Note, I am not one of the maintainers. I'm just a messenger._

**Comment by gyliu513** [comment]
Thanks @mattfarina for the explanation, I recalled that there are indeed exceptions for this, such as https://github.com/kubernetes/charts/pull/2059 , let's get some comments from @lachie83 as well ;-)

**Comment by gyliu513** [comment]
@lachie83 ping for comments

**Comment by gyliu513** [comment]
@lachie83 can we have this got merged?

**Comment by mattfarina** [comment]
There were some issues getting this to merge so I performed the squash and merge manually. This was merged via ea7406c0dc40c9adb4e4a3a10de481e0a41afbd8. Note, you're still listed as the committer.

**Pull request closed without merging**